packages/xrpl/src/models/methods/accountInfo.ts

```diff
@@ -180,3 +175,59 @@ export interface AccountInfoResponse extends BaseResponse {
     validated?: boolean
   }
 }
+
+/**
+ * Response expected from a {@link AccountInfoRequest}.
+ *
+ * @category Responses
+ */
+export interface AccountInfoResponse extends BaseAccountInfoResponse {
+  result: BaseAccountInfoResponse['result'] & {
+    /**
+     * Array of SignerList ledger objects associated with this account for Multi-Signing.
+     * Since an account can own at most one SignerList, this array must have exactly one
+     * member if it is present.
+     * Quirk: In API version 1, this field is nested under account_data. For this method,
+     * Clio implements the API version 2 behavior where is field is not nested under account_data.
+     */
+    signer_lists?: SignerList[]
+  }
+}
+
+/**
+ * Response expected from a {@link AccountInfoRequest} using API version 1.
+ *
+ * @category Responses
+ */
+export interface AccountInfoV1Response extends BaseAccountInfoResponse {
+  result: BaseAccountInfoResponse['result'] & {
+    /**
+     * The AccountRoot ledger object with this account's information, as stored
+     * in the ledger.
+     * If requested, also includes Array of SignerList ledger objects
+     * associated with this account for Multi-Signing. Since an account can own
+     * at most one SignerList, this array must have exactly one member if it is
+     * present.
+     */
+    account_data: BaseAccountInfoResponse['result']['account_data'] & {
+      /**
+       * Array of SignerList ledger objects associated with this account for Multi-Signing.
+       * Since an account can own at most one SignerList, this array must have exactly one
+       * member if it is present.
+       * Quirk: In API version 1, this field is nested under account_data. For this method,
+       * Clio implements the API version 2 behavior where is field is not nested under account_data.
+       */
+      signer_lists?: SignerList[]
+    }
+  }
+}
+
+/**
+ * Type to map between the API version and the response type.
+ *
+ * @category Responses
+ */
+export type AccountInfoVersionResponseMap<Version extends APIVersion> =
```

Comment:
I tried `api_version: 3` and rippled did throw an error `RippledError: invalid_API_version`. I don't think it's really worth adding a check since this is unlikely to happen and when it does it's a clear error from rippled.